utility/flight_stow.py

```diff
@@ -14,14 +14,37 @@ def startAnts(ants, dry_run=False):
         except:
             user_logger.warn("not all antennas were in 'STOP' mode.")
 
-def point_ants(ants, dry_run=False):
+def moveRI(ants, ridx_pos, dry_run=False):
```

Comment:
For Python code we follow the [pep8](https://www.python.org/dev/peps/pep-0008/) guidelines - function and variable names are all lowercase, with words separated by `_`.